client/src/components/common/applicant-info-3.vue

```diff
@@ -22,7 +22,7 @@
                         @input="mutateApplicant('emailAddress', $event)"
                         filled
                         hide-details="auto"
-                        placeholder="Email Address (for notifications)" />
+                        label="Email Address (for notifications)" />
```

Comment:
For some reason, the email rules on this field don't work like they do on p2.

Compare both pages. On p2, if you enter "sss" then the "Not a valid email" error displays. On p3, if you enter "sss" then the message disappears.

Reply:
let me check

Reply:
The phone number has the same problem.

It looks like the fields are validated when you leave them (onblur), instead of as you type. It's a bad user experience because you don't know if you have entered a valid value until you leave the field.

(It was broken before you got here, but please fix while you're here.)